R/build-home.R

```diff
@@ -120,6 +121,40 @@
 #'     href: http://website.com
 #' ```
 #'
+#' You can completely remove the sidebar.
+#'
+#' ```
+#' home:
+#'   sidebar: FALSE
+#' ```
+#'
+#' You can provide a ready-made sidebar HTML.
+#'
+#' ```
+#' home:
+#'   sidebar:
+#'     html: path-to-sidebar.html
+#' ```
+#'
+#' You can change the order of sidebar elements (except Dev Status):
+#' links, license (with an "s"), community, citation, authors
+#' and add custom ones.
+#' The example below creates a sidebar whose only elements will be the
+#' authors section, a custom section, and a Dev Status section if there are
+#' badges.
+#' Note that the text should be HTML formatted so if you add a link, you need a
+#' `<a>` tag.``
+#'
+#' ```
+#' home:
+#'   sidebar:
+#'     structure: [authors, custom]
+#'     components:
+#'       custom:
+#'         title: Funding
+#'         text: We are grateful for funding!
```

Comment:
It feels like `text` should be called `html`, and maybe it would be better to provide a `path` here, so that you can arbitrary sections in their own files?

Reply:
Should the two solutions co-exist? i.e. provide either path or html? I am just worried it'll look confusing.

Reply:
ok, the current state of the PR assumes inline HTML :relieved: